homeassistant/helpers/entity_platform.py

```diff
@@ -177,14 +176,6 @@ async def _async_setup_platform(self, async_create_setup_task, tries=0):
 
             await asyncio.wait_for(asyncio.shield(task), SLOW_SETUP_MAX_WAIT)
 
-            # Block till all entities are done
-            if self._tasks:
-                pending = [task for task in self._tasks if not task.done()]
-                self._tasks.clear()
-
-                if pending:
-                    await asyncio.gather(*pending)
```

Comment:
We are no longer waiting here. This code is common for all ways of starting platform setup.

Reply:
Oh. You're right. Sorry. Let me reinstate this code for now.